common/technology/btr_minor_races_species_tech.txt

```diff
@@ -0,0 +1,51 @@
+### Parnack Regime Tech
+
+btr_tech_parnkreg_regAlpha = {
```

Comment:
I have never seen camel case in any paradox file, so this might cause some problems.

Reply:
It worked so far, but I will make it an underscore to be safe